### PR TITLE
Provide more information about louis-db in the README.md

### LINKED ISSUE #28: Provide more information about louis-db in the README.
The README should be able to inform an external party of what louis-db is and does.

### PR DESCRIPTION
**Pull request by melanie-fressard**
Fixes #28 - The current description of this project is not satisfying, we need to improve that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# What is louis-db ?
+
+Louis-db is the database of [Louis](https://github.com/ai-cfia/louis). It includes all of the python and bash scripts as well as sql functions. It uses ada-002 API (Application Programming Interface) for semantic search.
```
Comment (k-allagbe) [inline README.md:3]: It's not very clear what ada-002 is.

File: database-schema.png
Comment (k-allagbe) [inline database-schema.png:1]: The `documents` model is not attached to anything?

**Review by k-allagbe — CHANGES_REQUESTED** [review]
The README explains better what this repository is.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+- The *louis* folder : This folder contains python files that allows connections to the database or the api.
+
+- The *postgres* folder : This folder contains the bash script that will set up the docker container.
+
+- The *sql* folder : This folder holds all of the sql functions and scripts.
+
+- The *tests* folder : This is the test folder, it allows you to test the code.
+
+Here is the database schema :
+![database schema](database-schema.png)
```
Comment (JolanThomassin) [inline README.md:16]: I feel like the schema is missing some tables

Comment (melanie-fressard) [reply README.md:16]: I agree, however I'm not sure if it's because we add table when we do our code

Comment (JolanThomassin) [reply README.md:17]: I don't remember adding tables

**Review by JolanThomassin — APPROVED** [review]

**Review by rngadam — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
+# What is louis-db ?
+
+Louis-db is the database of [Louis](https://github.com/ai-cfia/louis). It includes all of the python and bash scripts as well as sql functions. It uses ada-002 API (Application Programming Interface) for semantic search. 
+
+- The *bin* folder : This folder contains all of the bash scripts that would be useful to set up the database.
+
+- The *louis* folder : This folder contains python files that allows connections to the database or the api.
```
[the remark this answers is not in the document]
Comment (melanie-fressard) [reply README.md:8]: you're right, correcting that

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (melanie-fressard) [reply README.md:17]: done, I just wonder if it's ok some tables don't have any link on the schema

[the remark this answers is not in the document]
Comment (rngadam) [reply README.md:17]: some of these are views, other tables don't really have a relationship, so yes normal.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+- The *postgres* folder : This folder contains the bash script that will set up the docker container.
+
+- The *sql* folder : This folder holds all of the sql functions and scripts.
+
+- The *tests* folder : This is the test folder, it allows you to test the code.
+
+Here is the database schema :
+![database schema](database-schema.png)
+
+If you need to set up the database locally, please follow [this procedure](https://dev.azure.com/CFIA-DevOps-ACIA/AI-Lab/_wiki/wikis/AI-Lab.wiki/1039/Procedure-to-set-up-the-database-locally).
```
Comment (rngadam) [inline README.md:19]: Move this procedure to this repository. we don't generally want to link to the internal wiki from public doc.

**Review by rngadam — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (rngadam) [reply README.md]: @melanie-fressard typically you have to let the reviewer resolve the discussion thread

**Review by rngadam — APPROVED** [review]

**Pull request merged**